client/components/phone-input/country-flag.jsx

```diff
@@ -45,14 +45,17 @@ export default class extends React.Component {
 
 	renderFlag = () => {
 		const style = this.state.ready ? {} : { visibility: 'hidden' };
+		const { countryCode } = this.props;
 
-		if ( this.props.countryCode ) {
+		if ( countryCode ) {
 			if ( ! this.state.error ) {
+				const flagSvg = require( `flag-icon-css/flags/4x3/${ countryCode }.svg` );
```

Comment:
@ockham I'm afraid the assumption here that we have flags for every country on the list broke the checkout form :( Steps to reproduce:
 * Go to Domains -> Add -> Add any random domain name.
 * On the Domain Contact form, from the input phone field, select `Kosovo` as the country.

The screen will go blank and there will be an `Uncaught Error: Cannot find module './kv.svg'` in the console :(

Reply:
Ouch. Good find. I'll work on a fix.